specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2589,7 +2589,22 @@
       },
       "type": "object"
     },
+    "ActionPropertiesBase": {
+      "description": "Action property bag base.",
+      "properties": {
+        "logicAppResourceId": {
+          "description": "Logic App Resource Id, providers/Microsoft.Logic/workflows/{WorkflowID}.",
```

Comment:
The format description in the description does not seem to be complete given the value shown in the example files.
/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.Logic/workflows/MyAlerts

Please update it to use full value starts with /subscriptions

Reply:
Everything else looks good. Once updated the description, I will kick off the merge. Thanks

Reply:
I've change the description to be consist with the example file